cocotb/handle.py

```diff
@@ -202,12 +232,15 @@ def _discover_all(self):
 
         self._discovered = True
 
-    def _child_path(self, name):
-        """Returns a string of the path of the child :any:`SimHandle` for a given *name*."""
+    def _child_path(self, name) -> str:
+        """Return a string of the path of the child :any:`SimHandle` for a given *name*.
+
+        :meta public:
```

Comment:
This one strikes me as private, it's not useful to the user who can just use `obj.some_child.path`